src/IntercomCompanies.php

```diff
@@ -77,6 +77,39 @@ public function getCompany($id, $options = [])
         return $this->client->get($path, $options);
     }
 
+
+    /**
+     * Returns a list of Users belonging to a single Company based on the Intercom ID.
+     *
+     * @see    https://developers.intercom.com/reference#list-company-users
+     * @param  string $id
+     * @param  array  $options
+     * @return stdClass
+     * @throws Exception
+     */
+    public function getCompanyUsers($id, $options = [])
+    {
+        $path = $this->companyUsersPath($id);
+        return $this->client->get($path, $options);
+    }
+
+    /**
+     * Returns a list of Users belonging to a single Company based on the Company ID.
+     *
+     * @see    https://developers.intercom.com/reference#list-company-users
+     * @param  string $id
+     * @param  array  $options
+     * @return stdClass
+     * @throws Exception
+     */
+    public function getCompanyUsersByCompanyId($id, $options = [])
```

Comment:
I think this class already offers the ability to do this filtering if I'm not mistaken: `$intercomClient->companies->getCompanies(['type' => 'user', 'company_id' => $id])`

If that's the case, I think the idea behind this SDK is to act as an abstract wrapper around the API, but not really dealing with query params that make requests more specific. If the `getCompanies` method allows to retrieve this, I'd prefer to keep that method as the only path to do this, therefore not including this method in the SDK. What do you think?

Reply:
Ahh true that makes sense and definitely see where you're coming from. The only weirdness is that you'll be getting a list of users with the `getCompanies` method which feels a bit weird to me. But I'll remove this method for now and add that way to the read me as well 👍

Reply:
Oh, that's a great point, I didn't realize that in that case `getCompanies` would be returning users 🤔 I think that's a compromise we can take considering the goal of this SDK is only to access the intercom api endpoints being very lightweight